tests/11-custom-extractors.test.js

```diff
@@ -0,0 +1,63 @@
+const postcss = require('postcss')
+const fs = require('fs')
+const path = require('path')
+
+function run(input, config = {}) {
+  jest.resetModules()
+  const tailwind = require('../src/index.js')
```

Comment:
I've noticed that the second test in that file would pass even when it shouldn't, as long as the first one passed. I think it's due to this module sharedState which survives across multiple call in the same process. 

That's why I'm calling [`resetModules`](https://jestjs.io/docs/jest-object#jestresetmodules), which resets `require`'s cache and gives me a fresh new instance on each run.